pkg/skaffold/schema/errors/errors.go

```diff
@@ -200,3 +200,19 @@ func ConfigUnknownAPIVersionErr(version string) error {
 			},
 		})
 }
+
+// SkaffoldConfigUpgradeErr specifies that skaffold config needs to be upgraded to the latest version.
+func SkaffoldConfigUpgradeErr(version string) error {
+	msg := fmt.Sprintf("skaffold config version is %s not *latest.SkaffoldConfig", version)
```

Comment:
I think that the error message should say that `skaffold cannot auto-upgrade the config from version %s to version %s`. And the suggestion text should be similar to `upgrade skaffold config version to %s manually. Refer to the schema documentation at https://skaffold.dev/docs/references/yaml/`